scripts/setup.cfg

```diff
@@ -0,0 +1,3 @@
+[yapf]
```

Comment:
This file should be in root directory.

Reply:
I put it in in the project directory. Hopefully that is what you mean by root.